tool.py

```diff
@@ -0,0 +1,7 @@
+from jinja2 import Environment, PackageLoader, meta
```

Comment:
oo what's this do?!

Reply:
Its just what I found to look at variables in templates. Tho it doesn't like our templates.